tests/view/observer/mouseobserver.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals document */
+/* bender-tags: view, browser-only */
+
+import MouseObserver from 'ckeditor5/engine/view/observer/mouseobserver.js';
+import ViewDocument from 'ckeditor5/engine/view/document.js';
+
+describe( 'MouseObserver', () => {
+	let viewDocument, observer;
+
+	beforeEach( () => {
+		viewDocument = new ViewDocument();
```

Comment:
You need to destroy the viewDocument in afterEach(). Otherwise it stays listening the events what affects other tests.